packages/compass-aggregations/src/components/pipeline-toolbar/pipeline-header/pipeline-actions.tsx

```diff
@@ -69,14 +76,15 @@ export const PipelineActions: React.FunctionComponent<PipelineActionsProps> = ({
         Run
       </Button>
       <Link
+        aria-label={optionsLabel}
         as="button"
         className={optionsButtonStyles}
         data-testid="pipeline-toolbar-options-button"
         hideExternalIcon={true}
         onClick={() => onToggleOptions()}
       >
         <div className={optionStyles}>
-          {isOptionsVisible ? 'Less' : 'More'} Options{' '}
+          {optionsLabel}
           <Icon glyph={optionsIcon} />
         </div>
       </Link>
```

Comment:
This should have accordion roles https://www.w3.org/TR/wai-aria-practices-1.1/examples/accordion/accordion.html